### PR TITLE
Login(Token and JWT Authentication)

### PR DESCRIPTION
**Pull request by Nirav-1999**

### CODE REVIEW

File: BEProjectsApp/urls.py [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
 from BEProjectsApp import views
 
+
 app_name = "BEProjectsApp"
 router = DefaultRouter()
 router.register(r"projects", views.ProjectViewSet)
 router.register(r"teachers", views.TeacherViewSet)
 router.register(r"contributors", views.ContributorViewSet)
 
+# router.register(r"users", views.UserViewSet)
```
Comment (pujanm) [inline BEProjectsApp/urls.py]: Remove comments if not required.

**Comment by pujanm** [comment]
Otherwise, things look good to me!

File: BEProjectsApp/urls.py (hunk 1 above)
Comment (Nirav-1999) [reply BEProjectsApp/urls.py]: ok

**Comment by pujanm** [comment]
@Nirav-1999 there is an issue with the code.
Please look into this error: `ModuleNotFoundError: No module named 'rest_framework_jwt'`.
The build is failing because of this.

**Pull request merged**